packages/grafana-e2e/src/flows/addDashboard.ts

```diff
@@ -64,10 +63,12 @@ export const addDashboard = (config?: Partial<AddDashboardConfig>) => {
 
     fullConfig.variables = addVariables(variables);
 
-    e2e.components.BackButton.backArrow().click();
+    e2e.components.BackButton.backArrow().should('be.visible').click({ force: true });
   }
 
-  setDashboardTimeRange(timeRange);
+  if (timeRange) {
```

Comment:
Is this if statement necessary? Seems like it's always set?

Reply:
I was thinking of making it more flexible so it's not needed but thinking about it more, I'll remove it this 👍